pkg/k8s/watchers/watcher.go

```diff
@@ -421,7 +422,7 @@ var ciliumResourceToGroupMapping = map[string]watcherInfo{
 	synced.CRDResourceName(v2alpha1.BGPPName):     {skip, ""}, // Handled in BGP control plane
 	synced.CRDResourceName(v2alpha1.LBIPPoolName): {skip, ""}, // Handled in LB IPAM
 	synced.CRDResourceName(v2alpha1.CNCName):      {skip, ""}, // Handled by init directly
-
+	synced.CRDResourceName(v2alpha1.CCGName):      {skip, ""}, // Handled by CNP
```

Comment:
nit: Why is this marked as `skip` whereas both `CNPName` and `CCNPName` have entries here? 

```suggestion
	synced.CRDResourceName(v2alpha1.CCGName):      {start, k8sAPIGroupCiliumCIDRGroupV2Alpha1},
```

Reply:
Good catch, this was a leftover before the change that introduced the `sync.Once` for the shared CNP/CCNP/CCG handler. I'll fix it :+1: